docs/src/main/sphinx/language/sql-support.rst

```diff
@@ -0,0 +1,157 @@
+SQL statement support
+=====================
+
+The SQL statement support in Trino can be categorized into several topics. Many
+statements are part of the core engine and therefore available in all use cases.
+For example, you can always set session properties or inspect an explain plan
+and perform other actions with the :ref:`globally available statements
+<sql-globally-available>`.
+
+However, the details and architecture of the connected data sources can limit
+some SQL functionality. For example, if the data source does not support any
+write operations, then a :doc:`/sql/delete` statement cannot be executed against
+the data source.
+
+Similarly, if the underlying system does not have any security concepts, SQL
+statements like :doc:`/sql/create-role` can not be supported by Trino and the
+connector.
+
+The categories of these different topics are related to :ref:`read operations
+<sql-read-operations>`, :ref:`write operations <sql-write-operations>`,
+:ref:`security operations <sql-security-operations>` and :ref:`transactions
+<sql-security-transactions>`.
+
+Details of the support for specific statements is available with the
+documentation for each connector.
+
+.. _sql-globally-available:
+
+Globally available statements
+-----------------------------
+
+The following statements are implemented in the core engine and available with
+any connector:
+
+* :doc:`/sql/call`
+* :doc:`/sql/deallocate-prepare`
+* :doc:`/sql/describe-input`
+* :doc:`/sql/describe-output`
+* :doc:`/sql/execute`
+* :doc:`/sql/explain`
+* :doc:`/sql/explain-analyze`
+* :doc:`/sql/prepare`
+* :doc:`/sql/reset-session`
+* :doc:`/sql/set-session`
+* :doc:`/sql/set-time-zone`
+* :doc:`/sql/show-functions`
+* :doc:`/sql/show-session`
+* :doc:`/sql/use`
+* :doc:`/sql/values`
+
+.. _sql-read-operations:
+
+Read operations
+---------------
+
+The following statements provide read access to data and meta data exposed by a
+connector accessing a data source. They are supported by all connectors:
+
+* :doc:`/sql/select` including :doc:`/sql/match-recognize`
+* :doc:`/sql/describe`
+* :doc:`/sql/show-catalogs`
+* :doc:`/sql/show-columns`
+* ``SHOW CREATE MATERIALIZED VIEW``
+* :doc:`/sql/show-create-schema`
+* :doc:`/sql/show-create-table`
+* :doc:`/sql/show-create-view`
+* :doc:`/sql/show-grants`
+* :doc:`/sql/show-roles`
+* :doc:`/sql/show-schemas`
+* :doc:`/sql/show-tables`
+* :doc:`/sql/show-stats`
+
+.. _sql-write-operations:
+
+Write operations
+----------------
+
+The following statements provide write access to data and meta data exposed by
+by a connector accessing a data source. Availability varies widely from
+connector to connector:
+
+.. _sql-write-data:
+
+Data management
+^^^^^^^^^^^^^^^
+
+* :doc:`/sql/insert`
+* :doc:`/sql/update`
+* :doc:`/sql/delete`
+
+.. _sql-write-materialized-views:
+
+Materialized views
+^^^^^^^^^^^^^^^^^^
+
+* ``CREATE MATERIALIZED VIEW``
+* ``DROP MATERIALIZED VIEW``
+* ``REFRESH MATERIALIZED VIEW``
+
+.. _sql-write-schema-table:
```

Comment:
I'd name the anchor `sql-schema-table-management` or such

Reply:
done